ui/jstests/test-learning-resource.jsx

```diff
@@ -261,7 +269,15 @@ define(['QUnit', 'jquery', 'reactaddons', 'lodash', 'learning_resources',
           });
           var saveButton = $node.find("button")[0];
           React.addons.TestUtils.Simulate.click(saveButton);
+          assert.equal(
```

Comment:
It is ok to test loader in save LR panel only, because the show/hide mechanism is same in both cases success and failure.